content/lessons/chapter-6/put-it-together-5.tsx

```diff
@@ -154,6 +154,12 @@ const public_key_yirutads = ec.keyFromPublic(wit_key_ttefkgdk);
 console.log(public_key_yirutads.verify(hashed_message_bytes_aieoprds, wit_sig_okdakljl) && 'true');
 console.log("KILL")
   `,
+    rangesToCollapse: [
+      {
+        start: 115,
+        end: 136,
+      },
+    ],
```

Comment:
since the range is able to be anything as this will be collapsing user-generated code we will need to make this a number created from a regex of some kind

Reply:
That one slipped past me :).  Was too focussed on trying to get the collapsing to work in the editor.  Should be fixed now.